_source/logzio_collections/_tracing-sources/opentelemetry.md

```diff
@@ -1,29 +1,36 @@
 ---
-title: Ship traces with OpenTelemetry
+title: Installing the OpenTelemetry Collector for Distributed Tracing
 logo:
   logofile: opentelemetry-icon-color.png
   orientation: vertical
 data-source: OpenTelemetry
-description: How to deploy an OpenTelemetry Collector for traces to logz.io
+description: How to deploy an OpenTelemetry Collector for traces to Logz.io
 open-source:
   - title: Logz.io-OpenTelemetry trace exporter
     github-repo: https://github.com/open-telemetry/opentelemetry-collector-contrib
 contributors:
   - yyyogev
   - yberlinger
+  - doron-bargo
 shipping-tags:
   - traces
 ---
+## Overview
 
 Logz.io's trace exporter for OpenTelemetry allows you to ship distributed traces to Logz.io from different APM (Application Performance Management/Monitoring) agents, such as Jaeger, Zipkin, and so on.
 
+This topic explains how to install the OpenTelemetry Collector. For an overview of the process to send traces to Logz.io, see <a href="/user-guide/distributed-tracing/getting-started-tracing" target ="_blank"> Getting started with Logz.io Distributed Tracing</a>. 
+
+### OpenTelemetry components
+
 The OpenTelemetry Collector pipeline has the following main components: 
 
-* Receivers
-* Processors 
-* Exporters
+* Receivers are used to gather data from your environment (pull-based) or receive it from external sources (push-based). Your system must have at least one receiver configured. Receivers are typically HTTP/gRPC endpoints or daemon-like processes. 
 
-OpenTelemetry also includes extentions for additional functionality such as diagnostics and health checks, as well as a dedicated collector for components contributed by the community, such as the Logz.io exporter.
+* Processors are used at various stages of your pipeline to pre-process data before exporting it.  Processors may be used to modify attributes or sample the data. This component is used to ensure that your data successfully makes it through a pipeline.  <!-- how do you control what processors do? -->
+* Exporters are used to send data to your designated tracing (or metrics, or logging) backends/destinations. Your system must have at least one exporter configured to move data from collector to backend.
+
+OpenTelemetry also includes extensions for additional functionality, such as diagnostics and health checks for components contributed by the community, such as the Logz.io exporter, as well as a dedicated collector.
 
 #### Deploy OpenTelemetry Collector with Logz.io Exporter
 
```

Comment:
@yyyogev  - @afishler asks about version. Do we need to guide users to a specific version that we've tested? 

> Was it latest or specific version?